content/java/concepts/collection/collection.md

```diff
@@ -30,19 +30,19 @@ Where `DataType` is the data type to be stored in the collection and `Collection
 
 The `Collection` interface utilizes the following methods:
```

Comment:
@SSwiniarski @Dusch4593 @lucwhite My comment regarding the code formatting was in reference to the entire table, not just the description text. I believe that we generally do not use inline code in these tables and often the 'method' column are all links (if applicable). If this is not a formatting standard we're using or if this has changed recently please advise, so that I may address these elements properly in my own entries.

Reply:
To be honest, this is the first I've heard of this being a problem. I've had a number of PRs with code formatting in markdown tables to date. And to me, not doing the method names would seem odd.